frontend/src/api/SampleData.js

```diff
@@ -120,7 +129,8 @@ export function getSampleSender() {
     "Sender 1",
     "Online",
     sampleOutputChannels,
-    extras
+    extras,
+    "encodr",
```

Comment:
Typo & order and comma

Reply:
OUF thanks for catching that! fixed 9ea91ea